app/src/main/assets/chart.html

```diff
@@ -15,12 +11,13 @@
     <tr>
       {% for tile in tileRow %}
```

Comment:
OK, I know this is a long therm thing. I just wanted to know your thoughts on this. What about the <html> tag I mentioned here: https://github.com/projectbuendia/client/pull/35#discussion_r40755743 ?